src/views/splash.html

```diff
@@ -0,0 +1,108 @@
+                        </div>
+                    </div>
+                    <div id="intro-social" class="one unit column">
+                        <h4>
+                            Social
+                        </h4>
+                        <div>
+                            <a rel="external" href="https://www.facebook.com/hackcambridge">
+                                <img id="splash-facebook-logo" src="{{ asset('images/hc-2018/facebook-logo.svg') }}" alt="Facebook logo" draggable="false"/>
+                            </a>
+                            <a rel="external" href="https://twitter.com/@hack_cambridge">
+                                <img id="splash-twitter-logo" src="{{ asset('images/hc-2018/twitter-logo.svg') }}" alt="Twitter logo" draggable="false"/>
+                            </a>
+                            <a rel="external" href="https://medium.com/@HackCambridge">
+                                <img id="splash-medium-logo" src="{{ asset('images/hc-2018/medium-logo.svg') }}" alt="Medium logo" draggable="false"/>
+                            </a>
+                            <a rel="external" href="https://www.instagram.com/hack_cambridge/">
+                                <img id="splash-instagram-logo" src="{{ asset('images/hc-2018/instagram-logo.svg') }}" alt="Instagram logo" draggable="false"/>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="splash-signup" class="red">
+        <div class="container width">
+            <div class="one half unit row">
+                <div class="six unit column intro-collapses empty"></div>
+                <div class="seven unit column intro-collapses">
+                    <div class="one unit row">
+                        <div class="five unit column">
+                            <h5>
+                                Sign up now!<br><span class="secondary">We’ll be in touch soon.</span>
+                            </h5>
+                        </div>
+                    </div>
+                    <div class="half unit row">
+                        <div class="five unit column">
+                            <form id="splash-signup-form" class="half unit row" method="post" action="some-action">
+                                <div class="four small unit column">
+                                    <input type="email" name="signup-email" required placeholder="Enter your email address." />
+                                </div>
+                                <div class="one unit column submit">
+                                    <input type="submit" name="submit" value="Submit." />
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</main>
+<footer>
```

Comment:
Would it be possible to include the entire footer element in the `footer.html` template, or is it not generic enough?

Reply:
So the width of the container depends on what's going on further up on the page. In principle it might be possible to use variables to determine the class names, you'd still need to state the number of columns manually. So essentially the problem is unpredictable html needed to determine the structure. It might be possible to transition to CSS grid layout (http://caniuse.com/#search=grid%20layout) to get around problems like these.

Reply:
I should say I do intend to transition to CSS grid layout if it's feasible (will need to investigate this—I think I must have misread 'Can I use…' the last time I checked). However this will take some work on `grid.styl`, so it seems like something for a new pull request to me. Regarding the footer, I'm happy to maintain 'different' footers for the moment—by nature the footer is never going to grow to be very big or complex.